Shared/mods/deathmatch/logic/luadefs/CLuaVector3Defs.cpp

```diff
@@ -705,3 +707,49 @@ int CLuaVector3Defs::Eq(lua_State* luaVM)
     lua_pushboolean(luaVM, false);
     return 1;
 }
+
+int CLuaVector3Defs::Lt(lua_State* luaVM)
+{
+    CLuaVector4D* pVector1 = NULL;
+    CLuaVector4D* pVector2 = NULL;
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pVector1);
+    argStream.ReadUserData(pVector2);
+
+    if (!argStream.HasErrors())
+    {
+        lua_pushboolean(luaVM, (pVector1->fX < pVector2->fX) && (pVector1->fY < pVector2->fY) && (pVector1->fZ < pVector2->fZ));
+        return 1;
+    }
+    else
+    {
+        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
+    }
+
+    lua_pushboolean(luaVM, false);
```

Comment:
Change this line to: `lua_pushnil(luaVM);`
Functions must return nil when bad arguments are given.